source/wiz/utility.py

```diff
@@ -501,6 +501,7 @@ def compute_file_name(definition):
     The file name should be in the form of::
 
         "foo.json"
+        "namespace-foo-0.1.0.json"
```

Comment:
so all our nuke plugins would end up being nuke-sickle-nuke-1.0.0 etc?
Its only a utility function, so i guess it would not matter as the install plugin can do whatever it wants.

Reply:
'nuke-sickle-1.0.0' you mean, but yeah this is the plan. I considered adding sub-folder but that will lead to many level and could slow the system,  so making the name unique seems a better way to go

Reply:
true, the module is called sicke_nuke, the package just sickle... that makes it a bit less bad to read.
Fine then